resources/puppetlabs/lein-ezbake/template/foss/ext/redhat/init.erb

```diff
@@ -131,6 +131,13 @@ rh_status_q() {
     rh_status >/dev/null 2>&1
 }
 
+reload() {
+    echo -n $"Reloading $prog: "
```

Comment:
https://github.com/puppetlabs/ezbake/pull/334/files/13da957138a585aa929bf79a7e9dcdf07158199c#diff-636dbe07137aa76779d2f7c2ab149076R67 and https://github.com/puppetlabs/ezbake/pull/334/files/13da957138a585aa929bf79a7e9dcdf07158199c#diff-636dbe07137aa76779d2f7c2ab149076R104 echo messages when starting and stopping, so this is consistent within the file.